vtex/omnishipping
# Fixed Postal Code emptying when typing

#### What is the purpose of this pull request?
Fixed Postal Code emptying when typing right after profile information sent updated `orderForm`

#### How should this be manually tested?
1. Add [items to cart](https://fernando--vtexgame1.myvtex.com/checkout/cart/add?&workspace=fernando&sku=31&qty=1&seller=1&sc=1)
2. Proceed to Shipping
3. Type postal code `22071060`
4. Postal Code should be set successfully without emptying part of it

#### Screenshots or example usage
|Before|After|
|--|--|
|![bad](https://user-images.githubusercontent.com/5608421/70741049-46e3f380-1cf9-11ea-83c2-f9b5e93a0c18.gif)|![good](https://user-images.githubusercontent.com/5608421/70741066-4b101100-1cf9-11ea-870c-8ae39ab216bf.gif)|

#### Types of changes
